Cargo.toml

```diff
@@ -18,12 +16,30 @@ version = "0.1.0"
 edition.workspace = true
 rust-version.workspace = true
 default-run = "fish"
+# see also doc_src/license.rst
+# GPLv2
+#   - src/**/*.{rs,c}
+#   primary license
+# GPLv2+
+#   - src/builtins/printf.rs
+# BSD
+#   - share/tools/create_manpage_completions.py
+#   - doc_src/python_docs_theme/
+# LGPLv2+
+#   - src/wgetopt.rs
+# MIT
+#   - share/completions/grunt.fish
+license = "GPL-2.0-only AND GPL-2.0-or-later AND BSD-3-Clause AND LGPL-2.0-or-later AND MIT"
+license-file = "doc_src/license.rst"
+repository = "https://github.com/fish-shell/fish-shell"
+homepage = "https://fishshell.com"
+readme = "README.rst"
 
 [dependencies]
 pcre2 = { git = "https://github.com/fish-shell/rust-pcre2", branch = "master", default-features = false, features = ["utf32"] }
 
 bitflags = "2.5.0"
-errno = "0.3.9"
+errno = "0.3.0"
```

Comment:
This is probably ok. I assume that at 0.x there are essentially no guarantees of stability, so bumping a crate might cause issues elsewhere in the distribution?

Reply:
Ack. Unsure if this will cause issues with any other packages in fedora, but speaking from experience, packaging is hard and brittle in general.

Reply:
Yeah, I think the openSUSE approach (don't bundle every library as its own package) has some merit